adam-core/src/main/scala/org/bdgenomics/adam/rdd/read/AlignmentRecordRDD.scala

```diff
@@ -87,8 +86,14 @@ private[adam] class AlignmentRecordArraySerializer extends IntervalArraySerializ
   }
 }
 
-object AlignmentRecordRDD {
+object AlignmentRecordRDD extends Serializable {
 
+  /**
+   * Builds an AlignmentRecordRDD for unaligned reads.
+   *
+   * @param rdd The underlying AlignmentRecord RDD.
+   * @return A new AlignmentRecordRDD.
+   */
   def unaligned(rdd: RDD[AlignmentRecord]): AlignmentRecordRDD = {
     AlignmentRecordRDD(rdd,
       SequenceDictionary.empty,
```

Comment:
This could call `AlignmentRecordRDD(rdd, sequences, recordGroupDictionary, None)` directly